pkgs/development/compilers/llvm/common/flang-rt/default.nix

```diff
@@ -0,0 +1,78 @@
+{
+  stdenv,
+  llvm_meta,
+  monorepoSrc,
+  src,
+  runCommand,
+  cmake,
+  libclang,
+  libllvm,
+  mlir,
+  ninja,
+  python3,
+  flang-unwrapped,
+  version,
+}:
+let
+  pname = "flang-rt";
+
+  src' =
+    if monorepoSrc != null then
+      runCommand "${pname}-src-${version}" { } (''
+        mkdir -p "$out"
+        cp -r ${monorepoSrc}/cmake "$out"/cmake
+
+        mkdir -p "$out/llvm"
+        cp -r ${monorepoSrc}/llvm/cmake "$out/llvm/cmake"
+        cp -r ${monorepoSrc}/llvm/utils "$out/llvm/utils"
+        cp -r ${monorepoSrc}/third-party "$out/third-party"
+
+        cp -r ${monorepoSrc}/flang-rt "$out"/flang-rt
+        cp -r ${monorepoSrc}/flang "$out"/flang
+        cp -r ${monorepoSrc}/runtimes "$out"/runtimes
+      '')
+    else
+      src;
```

Comment:
These can just sit inside like in #389873

Reply:
Done!